pkg/lib/aws/cloudwatch.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+)
+
+func (c *Client) DoesLogGroupExist(logGroup string) (bool, error) {
+	_, err := c.CloudWatchLogs().ListTagsLogGroup(&cloudwatchlogs.ListTagsLogGroupInput{
+		LogGroupName: aws.String(logGroup),
+	})
+	if err != nil {
+		if CheckErrCode(err, "ResourceNotFoundException") {
```

Comment:
It's weird that this error code is different from the bucket not found error code, just an observation, I'm not sure we need to do anything about it now